src/porepy/compositional/compositional_mixins.py

```diff
@@ -1222,106 +1217,102 @@ def dependencies_of_phase_properties(
         Otherwise they are depend on pressure, temperature and **independent** overall
         fractions.
 
-        """
-        dependencies = [self.pressure, self.temperature]
-        if self._has_unified_equilibrium:
-            dependencies += [
-                phase.extended_fraction_of[component]
-                for component in phase
-                if self.has_independent_extended_fraction(component, phase)
-            ]
-        elif self._has_equilibrium:
-            dependencies += [
-                phase.partial_fraction_of[component]
-                for component in phase
-                if self.has_independent_partial_fraction(component, phase)
-            ]
-        else:
+        Important:
+            This method must be overwritten in every flow problem which relies on e.g.,
```

Comment:
It is not clear to me what the surrogate factory is an example of. I.e., I don't know in which other cases this comment is important.

Reply:
In cases with more than 1 phase or component, which relies on external flash calculations or look-ups like the OBL approach. Will it help to add these examples?

It boils down to the user having to provide information to the framework, on which variables the fluid properties depend
(`rho(p,T,z), rho(p,h,z)` etc.) This information is used to evaluate those variables and feed them to the flash or the look-up.

Reply:
Please provide the gist of this explanation to the dostring.

Reply:
See `Example:` section in the docstring.
This will be covered by the tutorial from #1254, where we need to revisit this docstring and shorten it respectively.

Reply:
I modified the documentation slightly to make it fit with my understanding. @vlipovac Could you verify that what I wrote is correct, and if so resolve this discussion?